src/utils/__tests__/changelog.test.ts

```diff
@@ -705,6 +706,103 @@ describe('generateChangesetFromGit', () => {
         '- Fix the clacking sound on gear changes (#950) by @alice',
       ].join('\n'),
     ],
+    [
+      `should expand commits & prs with the magic ${BODY_IN_CHANGELOG_MAGIC_WORD}`,
+      [
+        {
+          hash: 'abcdef1234567890',
+          title: 'Upgraded the kernel',
+          body: SKIP_CHANGELOG_MAGIC_WORD,
+        },
+        {
+          hash: 'bcdef1234567890a',
+          title: 'Upgraded the manifold (#123)',
+          body: '',
+          pr: {
+            local: '123',
+            remote: {
+              number: '123',
+              author: { login: 'alice' },
+              milestone: '1',
+            },
+          },
+        },
+        {
+          hash: 'cdef1234567890ab',
+          title: 'Refactored the crankshaft',
+          body: '',
+          pr: {
+            remote: {
+              number: '456',
+              author: { login: 'bob' },
+              body: `This is important and we'll include the __body__ for attention. ${BODY_IN_CHANGELOG_MAGIC_WORD}`,
```

Comment:
Should this be expected in the output?

Reply:
Nope. This is a "trick case" where it still gets folded up in the associated milestone.

Should probably clarify the text to express this then.